crates/vfs/src/loader.rs

```diff
@@ -32,6 +32,9 @@ pub struct Directories {
 /// [`Handle`]'s configuration.
 #[derive(Debug)]
 pub struct Config {
+    /// Version number to associate progress updates to the right config
+    /// version.
+    pub version: usize,
```

Comment:
```suggestion
    pub version: u32,
```

I can't articulate why, but I prefer to use `u32` as goto "an integer type". It feels proper not to depend on pointer width by default.